app/routes/api/v1/warehouse/orders/dates/index.ts

```diff
@@ -0,0 +1,6 @@
+import { LoaderFunction } from "@remix-run/node";
+import { methodNotAllowed } from "~/helpers/app-helpers.server";
+
+export const loader: LoaderFunction = async () => {
+  methodNotAllowed();
+};
```

Comment:
Is this needed?

Reply:
That has been put there so it doesn't throw the default remix 404 error